club_leader_onboarding_protocol.md

```diff
@@ -1,30 +1,58 @@
 # Club Leader Onboarding Protocol
 
-The things we need to do to onboard a new club leader.
-
-- [ ] Add to Slack
-  - [ ] Give them tips to introduce themselves
-- [ ] Add to GitHub Leaders team
-- [ ] Add to Facebook private group
-  - [ ] Give them tips to introduce themselves
-- [ ] Talk about what the the [code of conduct][code-of-conduct] is and why it's
-      important
-
-[code-of-conduct]: https://github.com/hackclub/hackclub/blob/master/CONDUCT.md
-
-## Acceptance Email
+Process for onboarding (and accepting) club leaders (in order):
+
+## Application Process
+
+1. An applicant submits an application to Hack Club through
+   https://hackclub.io/apply. It's sent to the team as an email to
+   team@hackclub.io.
+2. If the application looks good, a team member responds to the applicant to set
+   up an interview call. They also add the application to the "Applications"
+   sheet in https://clubs.hackclub.io and update the application's status as the
+   application progresses (ex. the interview call gets scheduled).
+3. In the interview call the team member's goal is to get a good idea of the
+   applicant's leadership skills. Below are a few good questions to ask. The
+   team member should be taking notes. After the call is over, they should
+   upload their notes as a private gist and link them in the "Application" sheet
```

Comment:
I'm assuming you mean the "Applications" sheet:

> 
![image](https://cloud.githubusercontent.com/assets/654941/12225644/bf71b304-b7c3-11e5-90bb-efb515e8128a.png)